releases/8.0.0.md

```diff
@@ -445,7 +453,7 @@ JavaScript:
 
     ```
 
-1. Update the JavaScript files in your project:
+1. Update the JavaScript files in your project as follows. (Note that even projects without custom JavaScript may have been templated with a `media/js/reports/default.js` that needs updating.)
```

Comment:
Maybe it's too much here, but it seems like it's worth mentioning that this default.js file can be removed. Maybe something like:

```Note that even projects without custom JavaScript may have been templated with a `media/js/reports/default.js`. This file may be removed if it is not intentionally being used to override default.js in core arches. If it is not removed, it must be updated as well.```